test/render-gear.tests.js

```diff
@@ -0,0 +1,26 @@
+import renderGear from '../src/render-gear.js';
```

Comment:
this file name should be "render-gear.test.js" (instead of render-gear.tests.js)